TOMP-API.yaml

```diff
@@ -1840,32 +1951,27 @@ components:
             users:
               type: array
               items:
-                $ref: '#/components/schemas/user'
+                $ref: "#/components/schemas/user"
 
     planningOptions:
       type: object
       description: Available option matching the query. Optionally including asset information for claiming specific assets. The pricing is also included.
-      required:
```

Comment:
Nice finds these! These'll probably have to be replaced rather than removed, but let's leave that to @kushshr who is working on adding all the required fields for issue #165

Reply:
OK, thanks. I wasn't sure what to replace them with, so @kushshr sounds like the better candidate :-)